docs/release-notes/breaking-changes.md

```diff
@@ -12,6 +12,13 @@ Breaking changes can impact your Elastic applications, potentially disrupting no
 % Description and impact of the breaking change.
 % For more information, check [PR #](PR link).
 
+::::{dropdown} 'close.on_state_change.removed' defaults to false on Linux and MacOS.
```

Comment:
It's better to invert here and put the exception in the documentation instead of listing all the rest. Technically, we can Filebeat on BSD and perhaps something else too.

```suggestion
::::{dropdown} 'close.on_state_change.removed' defaults to `true` on Windows and `false` on the rest of the platforms.
```